x-pack/packages/kbn-elastic-assistant/impl/assistant_context/index.tsx

```diff
@@ -148,6 +148,12 @@ export const AssistantProvider: React.FC<AssistantProviderProps> = ({
     baseSystemPrompts
   );
 
+  // if basePrompt has been updated, the localstorage should be accordingly updated
+  // if it exists
+  useEffect(() => {
+    setLocalStorageSystemPrompts(baseSystemPrompts);
+  }, [baseSystemPrompts, setLocalStorageSystemPrompts]);
+
```

Comment:
Awesome! Thank you for catching and fixing this 🙂

Reply:
We actually had to remove this as this was overriding users custom System Prompts. We will reintroduce a fix for `8.10`+ that merges the new base prompts with whatever the user has.